README.md

```diff
@@ -16,4 +17,7 @@ _Pessimism_ is a public good monitoring service for Optimism bedrock chains _(BA
 [![GitHub pull requests by-label](https://img.shields.io/github/issues-pr-raw/base-org/pessimism)](https://github.com/base-org/pessimism/pulls)
 [![GitHub Issues](https://img.shields.io/github/issues-raw/base-org/pessimism.svg)](https://github.com/base-org/pessimism/issues)
 
+**Warning:**
+Pessimism is currently experimental and very much in development. It means Pessimism is currently not secure nor stable, code will change and builds can break over the coming months. If you come across problems, it would help greatly to open issues so that we can fix them as quickly as possible. Furthermore, there is no active Coinbase hackerone bug bounty program for the Pessimism service.
```

Comment:
```suggestion
Pessimism is currently experimental and very much in development. It means Pessimism is currently not secure nor stable, so code will change and builds can break over the coming months. If you come across problems, it would help greatly to open issues so that we can fix them as quickly as possible. Furthermore, there is no active Coinbase HackerOne bug bounty program for the Pessimism service.
```

Reply:
[d7b1c91](https://github.com/base-org/pessimism/pull/5/commits/d7b1c914af0a0bcfc5007b922e6fc63e4e3f107d)